styles/components/shared/infowindow.scss

```diff
@@ -44,3 +44,54 @@
     }
   }
 }
+
+.c-profile-tooltip {
+  position: absolute;
+  box-shadow: $box-shadow-big;
+  font-family: $font-family-1;
+  background-color: $white;
+  color: $charcoal-grey;
+
+  > .title {
+    font-family: $font-family-1;
+    font-size: $font-size-regular;
+    line-height: 1;
+    letter-spacing: -0.6px;
+    height: 32px;
+    padding: 12px 8px;
+    background-color: $egg-shell;
+  }
+
+  .value {
+    padding: 10px 8px 8px 8px;
+
+    > .title {
+      font-family: $font-family-1;
+      font-size: $font-size-small;
+      letter-spacing: -0.5px;
+      opacity: 0.6;
+      text-transform: uppercase;
+      line-height: 1;
+    }
+
+    > .data {
+      font-family: $font-family-1;
+      font-size: 18px;
+      letter-spacing: -0.9px;
+      line-height: 1;
+      padding-top: 2px;
+
+      span {
+        opacity: 0.6;
+        font-size: 16px;
+        letter-spacing: -0.8px;
+      }
+    }
+
+    &:not(:last-child) {
+      border-bottom: 1px solid #ebebeb;
+    }
+  }
+
```

Comment:
Please run this file through the SCSS linter, it has many errors. (in particular color literals should be avoided)